Kernel/Bus/USB/USBDescriptors.h

```diff
@@ -126,7 +126,7 @@ static_assert(AssertSize<USBSuperSpeedEndpointCompanionDescriptor, 6>());
 struct [[gnu::packed]] USBHubDescriptor {
     USBDescriptorCommon descriptor_header;
     u8 number_of_downstream_ports;
-    union {
+    union [[gnu::packed]] {
         u16 raw;
```

Comment:
But late to the party but would making this use u8s internally also solve the issue?
Also assuming that descriptor common ends in odd alignent this should be properly aligned, could be that this is something I skipped over in the discord read back, in that case feel free to ping me the details again

Reply:
Note that forced packing caused other issues in the past, and the specs usually line up nicely with their item sizes and assumed alignments